openerp/service/server.py

```diff
@@ -232,7 +232,12 @@ def close_socket(self, sock):
         sock.close()
 
 class ThreadedServer(CommonServer):
+    initial_threads = []
+    
     def __init__(self, app):
+        for t in threading.enumerate():
+            self.initial_threads.append(t)
```

Comment:
That doesn't make sense, at this point the only thread which should exist is the main thread (whose id we store) since we haven't spawned anything yet, and the point of waiting on non-daemon thread is to avoid cutting off threads being created by modules like mail_channel or the procurement wizard (which are *not* daemonified).